packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/Snapshot.class.js

```diff
@@ -0,0 +1,30 @@
+export default class NetApp {
+  constructor({ id, name, description, createdAt, type, status }) {
+    Object.assign(this, {
+      id,
+      name,
+      description,
+      createdAt,
+      type,
+      status,
+    });
+  }
+
+  isActive() {
+    return [
+      'available',
+      'manage_starting',
+      'manage_error',
+      'unmanage_starting',
+      'unmanage_error',
+    ].includes(this.status);
+  }
+
+  isInactive() {
+    return ['error'].includes(this.status);
+  }
+
+  isPending() {
+    return ['creating', 'deleting', 'restoring'].includes(this.status);
```

Comment:
may be the constant array can be defined as a constant to avoid array initialisation every time this method is called?